src/main/webapp/app/course/manage/overview/course-management-card.component.html

```diff
@@ -254,7 +254,7 @@ <h4 class="text-center no-exercises mt-3 fw-medium">{{ 'artemisApp.course.noExer
             <span class="d-none d-xl-inline">{{ 'artemisApp.courseOverview.menu.messages' | artemisTranslate }}</span>
         </a>
         <a
-            *ngIf="course.isAtLeastTutor"
+            *ngIf="course.timeZone || course.isAtLeastInstructor"
```

Comment:
We currently don't load the tutorialGroupsConfiguration here and to keep this PR small I decided to only use the timeZone here, which is also a prerequisite for tutorials